https://bugzilla.xfce.org/show_bug.cgi?id=11239

--- origsrc/xfce4-session-4.10.1/scripts/xinitrc.in.in	2013-05-05 10:33:55.000000000 -0500
+++ src/xfce4-session-4.10.1/scripts/xinitrc.in.in	2015-06-21 06:53:52.113870700 -0500
@@ -22,6 +22,12 @@ if test "x$DESKTOP_SESSION" = "x"; then
   export DESKTOP_SESSION
 fi
 
+# set XDG_CURRENT_DESKTOP so that Qt 5 applications can identify user set Xfce theme
+if test "x$XDG_CURRENT_DESKTOP" = "x"; then
+  XDG_CURRENT_DESKTOP="XFCE"
+  export XDG_CURRENT_DESKTOP
+fi
+
 # $XDG_CONFIG_HOME defines the base directory relative to which user specific
 # configuration files should be stored. If $XDG_CONFIG_HOME is either not set
 # or empty, a default equal to $HOME/.config should be used.
--- origsrc/xfce4-session-4.10.1/scripts/xinitrc.in	2013-05-05 10:34:05.000000000 -0500
+++ src/xfce4-session-4.10.1/scripts/xinitrc.in	2015-06-21 06:56:28.147184400 -0500
@@ -22,6 +22,12 @@ if test "x$DESKTOP_SESSION" = "x"; then
   export DESKTOP_SESSION
 fi
 
+# set XDG_CURRENT_DESKTOP so that Qt 5 applications can identify user set Xfce theme
+if test "x$XDG_CURRENT_DESKTOP" = "x"; then
+  XDG_CURRENT_DESKTOP="XFCE"
+  export XDG_CURRENT_DESKTOP
+fi
+
 # $XDG_CONFIG_HOME defines the base directory relative to which user specific
 # configuration files should be stored. If $XDG_CONFIG_HOME is either not set
 # or empty, a default equal to $HOME/.config should be used.
--- origsrc/xfce4-session-4.10.1/xfce.desktop.in	2013-05-05 10:33:55.000000000 -0500
+++ src/xfce4-session-4.10.1/xfce.desktop.in	2015-06-21 06:54:25.967669600 -0500
@@ -5,3 +5,4 @@ _Comment=Use this session to run Xfce as
 Exec=startxfce4
 Icon=
 Type=Application
+DesktopNames=XFCE
