--- origsrc/xfce4-session-4.5.91/engines/balou/Makefile.am	2008-10-11 07:25:10.000000000 -0500
+++ src/xfce4-session-4.5.91/engines/balou/Makefile.am	2008-10-31 15:26:56.694976900 -0500
@@ -30,10 +30,5 @@
 libbalou_la_LIBADD =							\
 	$(top_builddir)/libxfsm/libxfsm-4.6.la
 
-if HAVE_OS_CYGWIN
-libbalou_la_LIBADD +=							\
-	@XFCE_MCS_MANAGER_LIBS@
-endif
-
 libbalou_la_DEPENDENCIES =						\
 	$(top_builddir)/libxfsm/libxfsm-4.6.la
--- origsrc/xfce4-session-4.5.91/engines/mice/Makefile.am	2008-10-11 07:25:11.000000000 -0500
+++ src/xfce4-session-4.5.91/engines/mice/Makefile.am	2008-10-31 15:27:07.554351900 -0500
@@ -19,11 +19,6 @@
 libmice_la_LIBADD =							\
 	$(top_builddir)/libxfsm/libxfsm-4.6.la
 
-if HAVE_OS_CYGWIN
-libmice_la_LIBADD +=							\
-	@XFCE_MCS_MANAGER_LIBS@
-endif
-
 libmice_la_DEPENDENCIES =						\
 	$(top_builddir)/libxfsm/libxfsm-4.6.la
 
--- origsrc/xfce4-session-4.5.91/engines/simple/Makefile.am	2008-10-11 07:25:11.000000000 -0500
+++ src/xfce4-session-4.5.91/engines/simple/Makefile.am	2008-10-31 15:27:16.210601900 -0500
@@ -20,11 +20,6 @@
 libsimple_la_LIBADD =							\
 	$(top_builddir)/libxfsm/libxfsm-4.6.la
 
-if HAVE_OS_CYGWIN
-libsimple_la_LIBADD +=							\
-	@XFCE_MCS_MANAGER_LIBS@
-endif
-
 libsimple_la_DEPENDENCIES =						\
 	$(top_builddir)/libxfsm/libxfsm-4.6.la
 	
--- origsrc/xfce4-session-4.5.91/libxfsm/Makefile.am	2008-10-11 07:25:15.000000000 -0500
+++ src/xfce4-session-4.5.91/libxfsm/Makefile.am	2008-10-31 15:24:43.757476900 -0500
@@ -22,7 +22,7 @@
 if HAVE_OS_CYGWIN
 libxfsm_4_6_la_LDFLAGS +=						\
 	-no-undefined							\
-	$(LIBX11_LDFLAGS@
+	$(LIBX11_LDFLAGS)
 
 libxfsm_4_6_la_LIBADD =							\
 	$(LIBX11_LIBS)							\
